aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -151,5 +151,35 @@ Tensor quantized_clone(const Tensor& self) {
   return dst;
 }
 
+bool quantized_equal(const Tensor& self, const Tensor& other) {
```

Comment:
nit: right now we use naming convention: "xxxx_quant" for quantized impelementations. so should be "equal_quant"

Reply:
That's not the case for `quantized_{conv,relu,max_pool2d,clone,adaptive_avg_pool2d_out}`

Personally, I think the abbreviation `quant` is less clear than saying `quantized`. `quant` is a prefix that exists in many different words, and shortening it by 4 characters doesn't save much.

Additionally, I don't like trailing adjectives because that's not the word order in English. You don't say `equal quantized`, you say `quantized equal`. 

imo the convention should be quantized_xxxx

Reply:
it's mostly matching the existing conventions for cpu/cuda tensor type ids, xxx_cpu, and xxx_cuda

Reply:
Additionally, trailing adjectives make the names inconsistent with the convention of {sparse,mkldnn,fbgemm,cudnn}_XXXX